### ISSUE TITLE
Android build throws exception

### ISSUE THREAD

**Issue by EvoTeamDevelop** [not-fl3/macroquad#313]
Hi, followed the guide how to build macroquad application for Android: https://github.com/not-fl3/miniquad/#android
Used simple example from https://github.com/not-fl3/miniquad/#android

While testing the created build in emulator everything seems fine, but while running it on actual phone it crashes, when application starts and throws exception:
2021-11-05 20:15:20.011 11702-11702/? E/AndroidRuntime: FATAL EXCEPTION: main
```
    Process: rust.translator, PID: 11702
    java.lang.UnsatisfiedLinkError: Unable to load native library "/data/app/rust.translator-WZN_OdRJNureQTMPI9DZww==/lib/arm64/libtranslator.so": dlopen failed: cannot locate symbol "glAttachShader" referenced by "/data/app/rust.translator-WZN_OdRJNureQTMPI9DZww==/lib/arm64/libtranslator.so"...
        at android.app.NativeActivity.onCreate(NativeActivity.java:195)
        at android.app.Activity.performCreate(Activity.java:7458)
        at android.app.Activity.performCreate(Activity.java:7448)
        at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1286)
        at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:3409)
        at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:3614)
        at android.app.servertransaction.LaunchActivityItem.execute(LaunchActivityItem.java:86)
        at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:108)
        at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:68)
        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2199)
        at android.os.Handler.dispatchMessage(Handler.java:112)
        at android.os.Looper.loop(Looper.java:216)
        at android.app.ActivityThread.main(ActivityThread.java:7625)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:524)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:987)
```

Used Huawei P20/P30 Pro, Android version 9.
Please share if I need to provide something else, it is my first experience with macroquad and I am not very sure how I can help here.

**Comment by tirithen** [comment]
I also had a similar problem and @not-fl3 released a patch version 0.3.13 now that fixed the example I put in https://github.com/not-fl3/macroquad/issues/338 , maybe this is the same issue that you where having?

**Comment by not-fl3** [comment]
Yes, 99% it is the same thing. The fix: update macroquad version to `0.3.13` and use the latest cargo-quad-apk(or the docker image). 

Feel free to reopen if it is not!
